dxapp.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "dnanexus_nirvana_2.0.3",
+  "title": "dnanexus_nirvana_2.0.3",
+  "summary": "dnanexus_nirvana_2.0.3",
+  "tags:": ["Annotation"],
+  "properties": {
+    "github release": "v1.0.0"
+  },
+  "dxapi": "1.0.0",
+  "inputSpec": [
+    {
+      "name": "input_vcf",
+      "label": "Input VCF file",
+      "help": "A VCF file containing variants to be annotated",
+      "class": "file",
+      "optional": false,
+      "patterns": ["*.refseq_nirvana_203.vcf"]
+    }
+  ],
+  "outputSpec": [
+    {
+      "name": "output_json",
+      "label": "Output JSON.gz file",
+      "help": "A JSON.gz file containing a structured JSON representation of all annotation and sample information",
+      "class": "file",
+      "patterns": ["*.json.gz"]
+    }
+  ],
+  "runSpec": {
+    "timeoutPolicy": {
+      "*": {
+        "hours": 48
+      }
+    },
+    "interpreter": "bash",
+    "release": "16.04",
+    "distribution": "Ubuntu",
+    "file": "src/dnanexus_nirvana_2.0.3.sh"
+  },
+    "access": {
+    "project": "CONTRIBUTE",
+    "allProjects": "VIEW",
+    "network": [
+      "*"
+    ]
+  },
+  "regionalOptions": {
+    "aws:us-east-1": {
```

Comment:
Will still app run in eu-central-1 (frankfurt)?
Can we add that as an option now?
I think some of the reference files are only in us-east-1 ...

Reply:
Correct, refs are in US right now. We don't have a dedicated refs project set up in Frankfurt yet, so I haven't updated the links to these files to use eu. Could store them in the existing Frankfurt project temporarily for now, but would still need to update the project:file again later. Similarly I have not changed the region to eu as the ref data is stuck in US. They're too big to store with the app (~25G).

Reply:
I'm thinking that if we need to use this app for patient data, it will have to run in frankfurt...